templates/typescript_gapic/src/$version/$service_client.ts.njk

```diff
@@ -156,6 +156,8 @@ export class {{ service.name }}Client {
     }
     if (!opts.fallback) {
       clientHeader.push(`grpc/${this._gaxGrpc.grpcVersion}`);
+    } else {
+      clientHeader.push(`rest/${this._gaxGrpc.grpcVersion}`);
```

Comment:
Now that the value is in `clientHeader`, it will just automatically show up in the header emitted by the GAPIC (ie no further coding work needed to include the header)? Cool!

Reply:
Correct